src/components/Layout/Layout.tsx

```diff
@@ -167,6 +169,36 @@ export const Layout = ({
       menuButtonRef?.current?.focus();
     }
   };
+  let menu = <Menu
+    currentPlatform={currentPlatform}
+    rootMenuNode={rootMenuNode}
+    menuTitle="How Amplify works"
+    menuHref={{
+      pathname: `/[platform]`,
+      query: { platform: currentPlatform }
+    }}
```

Comment:
I think the `menuTitle` and `menuHref` can be removed now since `Menu` doesn't have those props anymore